.babelrc

```diff
@@ -1,5 +1,9 @@
 {
-    "presets": ["@babel/preset-env"],
+    "presets": [["@babel/preset-env",
+        {
+            "targets": "> 0.5%, last 2 versions, Firefox ESR, not dead"
```

Comment:
This query (which is equivalent to `"defaults"`) includes Opera Mini which, among others things, doesn't support WebGL.